tests/tests.py

```diff
@@ -9,14 +9,33 @@
 import subprocess as sp
 from pathlib import Path
 
-from snakemake.resources import DefaultResources
-
 sys.path.insert(0, os.path.dirname(__file__))
 
 from .common import *
 from .conftest import skip_on_windows, only_on_windows, ON_WINDOWS, needs_strace
 
-from snakemake.resources import GroupResources
+
+@skip_on_windows
+def test_slurm_mpi():
+    run(dpath("test_slurm_mpi"), slurm=True)
```

Comment:
If we need to see the log file of the failing job:
```suggestion
    run(dpath("test_slurm_mpi"), slurm=True, show_failed_logs=True)
```